build.gradle.kts

```diff
@@ -12,7 +12,6 @@ plugins {
 repositories {
     mavenCentral()
     google()
-    jcenter() // remove after trove4j resolved https://youtrack.jetbrains.com/issue/KT-44730
```

Comment:
This is still needed, no?

Reply:
Yeah, need not now.

Reply:
Oh good, they finally got it moved
https://youtrack.jetbrains.com/issue/IDEA-261387